webflow-hacks/form-hacks/11/11.ts

```diff
@@ -0,0 +1,18 @@
+// when the DOM is ready
+document.addEventListener('DOMContentLoaded', function () {
+  // selector constants
+  const SHOW_PAGE_URL_SELECTOR = '[fs-hacks-element="show-page-url"]';
+  const PAGE_URL_INPUT_SELECTOR = '[fs-hacks-element="page-url-input"]';
+  // get elements
+  const page_url = document.querySelector<HTMLDivElement>(SHOW_PAGE_URL_SELECTOR);
```

Comment:
Any reason why these variables are using snake_case instead of camelCase?

Reply:
No reason. Changing to camel case!